src/System.Net.Sockets/tests/FunctionalTests/ConnectExTest.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.Net.Test.Common;
+﻿    using System.Net.Test.Common;
```

Comment:
Thanks for noticing! Not really sure how this got in. Looking at other non-Security settings now.